src/main/java/org/influxdb/impl/InfluxDBImpl.java

```diff
@@ -287,4 +340,18 @@ public void deleteDatabase(final String name) {
     }
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public void close() {
+    try {
+        this.disableBatch();
```

Comment:
@majst01 Done with readme change and javadoc improve (for this method, it use {@inheritDoc} . the javadoc is located in interface method)
Thanks for your review